app/src/main/java/org/p2p/wallet/qr/ui/ScanQrFragment.kt

```diff
@@ -101,6 +100,10 @@ class ScanQrFragment :
         }
     }
 
+    override fun setSystemBarsStyle() {
```

Comment:
set means that we need to pass some parameter to `set` system bars, some style of other 
i liked the approach with fields more, can't we stick to it?